### PR TITLE
LPS-148738 baseline

### PR DESCRIPTION
**Pull request by marcelabc**
Related Issue: https://issues.liferay.com/browse/LPS-148738

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Review by gabrielwas — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectValidationRuleLocalServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+		try {
+			_testAddObjectValidationRule(
+				ObjectValidationRuleConstants.ENGINE_TYPE_DDM, "",
+				"isEmailAdress(textField)");
+
+			Assert.fail();
+		}
+		catch (ObjectValidationRuleException objectValidationRuleException) {
+			Assert.assertEquals(
+				"Label is null for locale " + LocaleUtil.US.getDisplayName(),
```
Comment (gabrielwas) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectValidationRuleLocalServiceTest.java:113]: @marcelabc _Label is null_ or _Name is null_ ?

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectValidationRuleServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+		}
+		catch (PrincipalException.MustHavePermission principalException) {
+			String message = principalException.getMessage();
+
+			Assert.assertTrue(
+				message.contains(
+					StringBundler.concat(
+						"User ", _defaultUser.getUserId(),
+						" must have UPDATE permission for")));
+		}
```
Comment (gabrielwas) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectValidationRuleServiceTest.java:135]: @marcelabc I think we can suppress this test here... _objectDefinition is enough

**Review by gabrielwas — CHANGES_REQUESTED** [review]

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:stop

**Comment by liferay-forms** [comment]
Hey @marcelabc, can you please check if these failures are related to your changes? Thanks

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @marcelabc, can you please check if these failures are related to your changes? Thanks

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @marcelabc, can you please check if these failures are related to your changes? Thanks

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**